### ISSUE TITLE
chest's items inside dont lb back

### ISSUE THREAD

**Issue by rockett1** [LogBlock/LogBlock#131]
when placing a chest adding items to it and filling up to the brim with items in the time period of about a week then someone goes to grief the building. after getting an admin to logblock back the damage to the house everything in the chests seems to dissapeared, gone no where it's really frustrating when you have earned all those items is there a way to get this plugin to save items in chests as this has become a real bug to my server and its community within.

so in short 
do chests logblock back with there items inside?
if so how come it doesn't work on my server?
is there an option that is needed to be enabled or is this a bug?

i will be eternally grateful for you to resolve this issue or if its not an issue please give an answer as i am fully aware you work hard into tweaking your plugins to perfection.
thanks,
